python-api/gstswitch/helpers.py

```diff
@@ -8,7 +8,23 @@
 from .exception import RangeError, InvalidIndexError
 
 
-__all__ = ["TestSources", "PreviewSinks"]
+__all__ = ["TestSources", "PreviewSinks", "assert_no_segfault"]
+
+
+def assert_no_segfault(serv):
+
+    """Test is a closed Server-Processed died because of a SEGMENTATION
+    FAULT and print its Log if it did
```

Comment:
This function should probably be on the serv object itself.

This function doesn't actually assert anything, only print messages.

It also kills serv if it is till running.